media-capabilities/index.js

```diff
@@ -17,31 +17,28 @@ import { useState, useEffect } from 'react';
 
 const supported = typeof window !== 'undefined' && 'mediaCapabilities' in navigator;
```

Comment:
I deliberately left it like that because I think the user of this hook only wants to know if it's supported or not. If the browser api is not available then we cannot test if the media is supported so we fallback to `false`.

But indeed, rewriting the api like this would make more sense
```
const { supported, mediaCapabilitiesInfo } = useMediaCapabilitiesDecodingInfo(mediaDecodingConfig)
```

Reply:
In my opinion, the fact that browser API is not available does not mean that the given content type cannot be decoded on the device.

So I'd rather choose the rewrite. Then perhaps we need to update the README accordingly (explaining return value).

Reply:
True, with the seperated `support` property the developer would have a clear way of checking if the browser api is supported or not.
API changed, README updated.